specs/vep-554-apache-airflow-integration/README.MD

```diff
@@ -187,58 +173,14 @@ by providing a sensor operator derived from `airflow.sensors.base.BaseSensorOper
 which polls on the specific time intervals for job execution completion and does not hold
 the Airflow Worker.
 
-* Availability.
-    * The availability of the VDK Provider will be managed by Airflow, since it is going
-      to be deployed as part of Airflow ecosystem.
-
-* Capacity Estimation and Constraints TODO
-
-<!--
-* Capacity Estimation and Constraints
-    * Cost of data path: CPU cost per-IO, memory footprint, network footprint.
-    * Cost of control plane including cost of APIs, expected timeliness from layers
-      above.
-
-* Performance.
-    * Consider performance of data operations for different types of workloads. Consider
-      performance of control operations
-    * Consider performance under steady state as well under various pathological
-      scenarios, e.g., different failure cases, partitioning, recovery.
-    * Performance scalability along different dimensions, e.g. #objects, network
-      properties (latency, bandwidth), number of data jobs, processed/ingested data, etc.
-* Telemetry and monitoring changes (new metrics).
-* Configuration changes.
-* Upgrade / Downgrade Strategy (especially if it might be breaking change).
-* Troubleshooting
-    * What are possible failure modes.
-        * Detection: How can it be detected via metrics?
-        * Mitigations: What can be done to stop the bleeding, especially for already
-          running user workloads?
-        * Diagnostics: What are the useful log messages and their required logging levels
-          that could help debug the issue?
-        * Testing: Are there any tests for failure mode? If not, describe why._
-* Operability
-    * What are the SLIs (Service Level Indicators) an operator can use to determine the
-      health of the system.
-    * What are the expected SLOs
-* Test Plan
```

Comment:
I am not seeing a test plan . 

Particularly are there any recommendations and practices that we follow for adding new provider. If there's link of Airflow doc, add it here. Or do we need to come up with something ourselves?

Reply:
We will follow the official Airflow documentation - https://airflow.apache.org/docs/apache-airflow-providers/howto/create-update-providers.html#initial-code-and-unit-tests